libraries/javax.servlet_3.1/library.xml

```diff
@@ -2,6 +2,7 @@
 	<name>Servlet API</name>
 	<description>Servlet API</description>
 	<version>3.1</version>
+	<release>0</release>
```

Comment:
What does that change?

Reply:
nothing, just to conform with the maven version